.gitignore

```diff
@@ -89,10 +88,12 @@ tags
 .vscode/*
 .history
 openshift.local.clusterup/*
-build/_output
 vendor/
+### JetBrains ###
 .idea/
-*.iml
+### Vagrant ###
```

Comment:
Why this needed for the project?

Reply:
initially started with a Vagrant env for dev but now I'm just kind :). It's not needed anymore, will remove it.